### PR TITLE
add suffix to model saving file

### PR DESCRIPTION
**Pull request by ja-nina**
add suffix and a higher lr for ViT(no plateau in training loss seen with previous)

### CODE REVIEW

**Comment by ja-nina** [comment]
I think it also would be of benefit to have a single model training script that then can be reused over and over again in a shell (to avoid main.py crashing) - but that''s for the future
